libs/cardano-ledger-binary/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 1.1.2.1
 
-*
+* Add `fieldGuarded` to be able to conditionally construct a `Field` #
```

Comment:
This is a semantic change, addition of a new function, which means it must be a minor bump: `1.1.3.0`

Also, I am sure you were gonna do this anyways, but just in case :wink: 
```suggestion
* Add `fieldGuarded` to be able to conditionally construct a `Field` #3712 
```